### PR TITLE
Adding using-pass.py in loops directory

### LINKED ISSUE #105: [tutorial] pass statement
Along with break and continue statements in loops, we also have pass statement. But it's not that common in use. It
is mainly used in production codes only.  But still it's good to know about it. 
**Description of the tutorial**

About 'pass'
An example
A challenge

...

**Do you want to be assigned to work on this?**
(mark [x])
- [x] YES
- [ ] NO

**Optional: Any additional info you want to add**
...

### PR DESCRIPTION
**Pull request by deepu14d**
### 2. Description 
Creating using-pass.py file inside loops directory. The files content give us introduction to pass statement with an example.  

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [x] My code is formatted using required linting
- [x] I've run my code locally and checked it works

### CODE REVIEW

**Comment by deepu14d** [comment]
Can you please also add the hacktoberfest label to this? 

> ### 1. This pull request resolves #105
> ### 2. Description
> Creating using-pass.py file inside loops directory. The files content give us introduction to pass statement with an example.
> 
> ### 3. Fill in checklist by marking [x]
> * [x]  I've read the CONTRIBUTING.md
> * [x]  I was assigned to this issue
> * [x]  My code is formatted using required linting
> * [x]  I've run my code locally and checked it works

File: loops/using-pass.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+while i < len(my_list):
+    print(my_list[i])
+    i += 1
+
+# In above example we won't get any error as we are using pass but if we don't use pass and just use
+# for loop without body then we'll get error.
+
+
+# ------------------------------------------------------------------------------------
+# Challenge: Create a function that asks user whether he/she uses github or not, if no then print the steps to make github account
```
Comment (sabderemane) [inline loops/using-pass.py:27]: I don't understand the challenge, what do you mean ? It sounds complicated for me.

Plus, you are asking the user to create a function and the user has not seen function yet.

**Review by sabderemane — CHANGES_REQUESTED** [review]
Thanks @deepu14d , I have suggested changes.

**Comment by deepu14d** [comment]
> Thanks @deepu14d , I have suggested changes.

Oh sorry
It seem confusing to me too at first. 
Can you please help in adding a challenge? I am unable to come with something good n easy (and I am travelling, don't have access to PC. Sorry) I'll try to change challenge ASAP.

File: loops/using-pass.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# and if yes then just pass it and print "that's good" outside the body of else block but inside the function.
+# Then call that function exactly 3 times using a for loop.
```
Comment (sabderemane) [inline loops/using-pass.py:29]:
```suggestion
# Challenge: Create a loop with fruits and number, if it's a fruit print it, and if it's a number pass it.
# note : use typeof to make the condition
```

**Review by sabderemane — COMMENTED** [review]
I have made a suggestion @deepu14d , tell me what you think

**Comment by deepu14d** [comment]
> I have made a suggestion @deepu14d , tell me what you think

Yeah yeah it's fine, better than mine challenge. 
Thanks

**Review by sabderemane — APPROVED** [review]

**Comment by sabderemane** [comment]
Thank you for your contribution @deepu14d 🎉

**Pull request merged**